docs/solution-test-plan.rst

```diff
@@ -0,0 +1,110 @@
+Solution Test Plan
+==================
+
+Introduction
+------------
+
+About this document
+^^^^^^^^^^^^^^^^^^^
+
+This document outlines the process for validating an Openstack deployment utilizing F5 products to provide Neutron LBaaSv2 (load balancing) services. Use cases are defined, for testing purposes, that encompass the set of standard F5 OpenStack solution deployments.
+
+F5 OpenStack Integrated Solutions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+F5 produces integration solutions that orchestrate BIG-IP Application Delivery Controllers (ADC) with OpenStack Networking (Neutron) services. F5 OpenStack LBaaSv2 integration provides under-the-cloud multi-tenant infrastructure L4-L7 services for Neutron tenants.
+
+In addition to community OpenStack participation, F5 maintains partnerships with several OpenStack platform vendors. Each vendor defines a certification process, including test requirements, that expand on or focus tests available in community OpenStack. This document presumes use of a certified deployment; to the extent vendor tests have been or will be run to prove the validity of the deployment.
+
+Community OpenStack and platform vendor tests exercise the generic LBaaSv2 integration. F5 OpenStack tests exercise F5-specific capabilities across multiple network topologies. They are complementary to community and platform vendor tests.
+
+All F5 OpenStack tests are available in the same open source repository as the product code. They may be executed via tempest and tox, consistent with the OpenStack community, to allow self-validation of a deployment.
+
+Use cases are based on real-world scenarios that represent repeatable deployments of the most common features used in F5 OpenStack integrations. Use case tests validate the combination of OpenStack, F5 BIG-IP ADC and F5 OpenStack products.
+
+Prerequisites
+-------------
+
+OpenStack
+^^^^^^^^^
+
+* Operational |openstack| OpenStack cloud deployed in accordance with minimal documented requirements:
+
+  * Deployment configuration will be varied to match test architectures described within each use case.
+  * 1 host machine for a Controller node
+  * 1 host machine for a Compute node
+
+* Nova flavor created to `accommodate <https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/bigip-ve-setup-linux-kvm-12-1-0.pdf>`_ the TMOS instance (link is for 12.1.0)
+
+TMOS
+^^^^
+
+* Supported TMOS |tmos_supported_versions|
+* For Virtual Edition
+
+  * LTM_1SLOT KVM qcow2 image built using the supported Onboarding Heat |heat_ve_onboard_template|
+  * Instance deployed using the supported 3-NIC Heat |heat_ve_3nic_template|
```

Comment:
I see what you're doing here. Sphinx will also complain about the same word ('template') being used for different target URLs. I'd use the intersphinx refs and use the template name as the text to display.
`:ref:`3-NIC Heat template <heat:F5® BIG-IP® VE: Standalone, 3-nic>`
Generally speaking, the title of the doc should work as the target.